packages/desktop-client/src/components/budget/BalanceWithCarryover.tsx

```diff
@@ -19,20 +22,30 @@ type BalanceWithCarryoverProps = {
 export default function BalanceWithCarryover({
   carryover,
   balance,
+  goal,
+  budgeted,
   disabled,
   style,
   balanceStyle,
   carryoverStyle,
 }: BalanceWithCarryoverProps) {
   let carryoverValue = useSheetValue(carryover);
   let balanceValue = useSheetValue(balance);
-
+  let goalValue = useSheetValue(goal);
+  let budgetedValue = useSheetValue(budgeted);
+  let isGoalTemplatesEnabled = useFeatureFlag('goalTemplatesEnabled');
```

Comment:
I think we can remove this check. Is `goalValue` going to be `null` when the feature flag is disabled?

Reply:
The idea was the if someone disabled the goal templates after trying it the colors would not show.  The goals don't go away in the db, but this will block them from being used.